### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_View Properties_Keyboard: Unable to access the Collapse/Expand button in the editor code using Keyboard

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2702]
[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#Azure Tools for VS Code; #A11ySev2; #A11yTCS; #BM-Azure Tools for VS Code-Win-Feb2021; #DesktopApp; #WCAG2.1.1;#Win10; #Keyboard;

**Environment Details:**
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0

OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### Repro Steps:

1. Open Visual studio code
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
4. Tab till "Install" button and press enter on it.
5. Tab till "Azure" button from left side pane and press enter on it.
6. Default screen will get open.
7. Tab to any of the list items and expand it using arrow keys.
8. Navigate to any sub list items and press enter on context menu.
9. Select "View Properties" using down arrow and press enter to select it.
10. Tab till Collapse button which are present in front of code lines in the editor and Check whether user is able to access collapse button using keyboard or not.

### Actual Result:
Unable to access the Collapse/Expand button in the editor code using Keyboard

### Expected Result:
User should be able to access Collapsed/Expand button in the editor code section using keyboard

### User Impact:
Keyboard only users will have difficulty in accessing the  button in the editor code section

### Attachment for Reference:
[2702_Keyboard_View Properties.zip](https://github.com/microsoft/vscode-azurefunctions/files/6028102/2702_Keyboard_View.Properties.zip)

**Comment by ejizba** [comment]
This is external to us - we don't control the editor. There's a decent number of existing issues like https://github.com/microsoft/monaco-editor/issues/2267, but I think this is the root issue: https://github.com/microsoft/vscode/issues/43262

Fwiw, there's a bunch of command palette commands and keyboard shortcuts already:
<img width="605" alt="Screen Shot 2021-03-19 at 4 14 38 PM" src="https://user-images.githubusercontent.com/11282622/111843288-e154be80-88ce-11eb-933a-b64befe513dd.png">
